code/web/src/modules/user/api/actions.js

```diff
@@ -66,6 +68,37 @@ export function login(userCredentials, isLoading = true) {
   }
 }
 
+export function updateProfileInfo(newProfileData) {
+  return dispatch => {
+
+    return axios.post(routeApi, mutation({
+      operation: 'userUpdate',
+      variables: newProfileData,
+      fields: ['id', 'name', 'email', 'bio', 'street', 'city', 'state', 'zip', 'image']
```

Comment:
Love how easily this hooked into the BE to update data!

Reply:
This is so cool!! Love seeing how the two intersect!